examples/nek5000/nek-examples.sh

```diff
@@ -0,0 +1,329 @@
+#!/bin/bash
+
+# Copyright (c) 2017, Lawrence Livermore National Security, LLC. Produced at
+# the Lawrence Livermore National Laboratory. LLNL-CODE-734707. All Rights
+# reserved. See files LICENSE and NOTICE for details.
+#
+# This file is part of CEED, a collection of benchmarks, miniapps, software
+# libraries and APIs for efficient high-order finite element and spectral
+# element discretizations for exascale applications. For more information and
+# source code availability see http://github.com/ceed.
+#
+# The CEED research is supported by the Exascale Computing Project (17-SC-20-SC)
+# a collaborative effort of two U.S. Department of Energy organizations (Office
+# of Science and the National Nuclear Security Administration) responsible for
+# the planning and preparation of a capable exascale ecosystem, including
+# software, applications, hardware, advanced system engineering and early
+# testbed platforms, in support of the nation's exascale computing imperative.
+###############################################################################
+# Script for Building and Running Nek5000 examples
+###############################################################################
+## Nek5000 path
+#NEK5K_DIR=
+
+## NEKTOOLS path
+#NEK5K_TOOLS_DIR=
+
+## CEED path
+#CEED_DIR=
+
+## Fortran compiler
+#FC=
+
+## C compiler
+#CC=
+
+###############################################################################
+# DONT'T TOUCH WHAT FOLLOWS !!!
+###############################################################################
+# Set defaults for the parameters
+: ${NEK5K_DIR:=`cd "../../../Nek5000"; pwd`}
+: ${NEK5K_TOOLS_DIR:=`cd "${NEK5K_DIR}/bin"; pwd`}
+: ${CEED_DIR:=`cd "../../"; pwd`}
+: ${FC:="mpif77"}
+: ${CC:="mpicc"}
+: ${MPI:=0}
```

Comment:
Set default to serial - Travis doesn't need to build w/mpifort

Reply:
This might surprise a user who wishes to experiment with Nek. I think I'd rather see Travis `export MPI=0` instead of hard-coding it here.

Reply:
Updated